postgres-connector/src/main/java/org/odpi/openmetadata/adapters/connectors/integration/postgres/ffdc/PostgresConnectorAuditCode.java

```diff
@@ -20,92 +20,45 @@
  */
 public enum PostgresConnectorAuditCode implements AuditLogMessageSet
 {
-    ERROR_READING_DATABASES("POSTGRES-CONNECTOR-0001",
+    ERROR_READING_POSTGRES("POSTGRES-CONNECTOR-0001",
             OMRSAuditLogRecordSeverity.EXCEPTION,
-            "Error Reading Databases from Server",
-            "Verify that the Database Server is available",
-            "Verify that the connector is being passed the correct connection properties"),
+            "An SQL Exception was received by method {0} from the postgres server",
+            "Verify that the postgres database is available",
+            "If the postres database is available then contact the egeria team for support"),
```

Comment:
typo - postres

But also I think the official wording is 'PostgreSQL' - I think we need to consider how we should refer to the technology in our docs/messages. I've seen 'Postgres' used but this is also capitalized

Reply:
egeria should probably be capitalized,